docs/reference/migration/migrate_5_0/suggest.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[breaking_50_suggester]]
+=== Suggester changes
+
+Now `completion` and `context` suggest fields are indexed as lucene's
+suggest fields. This means new `completion` indices are not compatible
+with pre-5.0 indices. Older `completion` indices can still be queried
+and indexed in the same way as before but not created. It is strongly
+recommended to reindex `completion` fields to take advantage of the
+following:
+
+==== Completion suggester is near-real time
+
+Previously, deleted suggestions could be included in results even
+after refreshing an index. Now, deletions are visible in near-real
+time, i.e. as soon as the index has been refreshed. This applies
+to suggestion entries for both context and completion suggesters.
+
+==== Completion suggester is document-oriented
+
+Suggestions are aware of the document they belong in. This enables
+retrieving any field value from the document. This is exposed
+through query-time `payload` option in `completion` and `context`
+suggesters:
+
+[source,sh]
+---------------
+GET /my_index/_search
+{
+  "suggest": {
+    "fooSuggestion": {
+       "text": "f"
+       "completion": {
+         "field": "fooSuggest",
+         "payload": ["field1", "field2"]
+       }
+     }
+  }
+}
+---------------
+
+Previously, `context` and `completion` suggesters supported index-time
+`payloads` option, which was used to return metadata with suggestions.
+Now metadata can be stored as a field in the same document as the
+suggestion for enabling retrieval at query-time. The support for
+index-time `payloads` has been removed to avoid bloating the in-memory
+index with suggestion metadata. Note that query-time `payloads` retrieval
```

Comment:
Perhaps delete the sentence "Note that..."  and replace it with: 

    The time that it takes to retrieve payloads depends heavily on the size of the `_source` field.  
    The smaller the `_source`, the faster the retrieval.